packages/open-next/src/build/helper.ts

```diff
@@ -73,6 +73,7 @@ function findMonorepoRoot(appPath: string) {
       { file: "yarn.lock", packager: "yarn" as const },
       { file: "pnpm-lock.yaml", packager: "pnpm" as const },
       { file: "bun.lockb", packager: "bun" as const },
+      { file: "bun.lock", packager: "bun" as const },
```

Comment:
It looks like `bun install --yarn` will also generate a `yarn` lock file so maybe the bun entries should be moved before yarn?